torch/distributed/_sharded_tensor/ops/equals.py

```diff
@@ -1,58 +1,7 @@
 import torch
```

Comment:
We can get rid of this file and just put everything in `binary_cmp.py`?

Reply:
done